ui/src/app/core/settings.service.ts

```diff
@@ -32,6 +32,9 @@ export class SettingsService {
   public browserLightingMode: 'light' | 'dark'
   public menuMode: 'default' | 'freeze'
   public wallpaper: string
+  public terminalPersistence = false
+  public terminalShowWarning = false
+  public terminalBufferSize = 50000
```

Comment:
We can access the globals file using `globalThis`, see here for an example:
- https://github.com/homebridge/homebridge-config-ui-x/blob/1b3df40391a4bd5d2eefc91727a1ccf2cfe78c73/ui/src/app/modules/settings/backup/backup.component.ts#L78

This is if you do end up putting that number into the file